package.json

```diff
@@ -126,8 +123,31 @@
     "yaml": "^1.10.0"
   },
   "workspaces": [
-    "configs/*",
-    "packages/*",
-    "scripts/docker"
+    "configs/eslint-config-mongosh",
```

Comment:
We'll generate this with a hook somewhere. To be decided exactly where. preinstall?

Reply:
Is there any chance we could just use the new monorepo-tools scripts instead of hardcoding a list here?

Reply:
We're now trying to avoid rewriting lerna since there's quite a bit of scope creep there.

Reply:
Plan would be adding "workspacePatterns" (which contains what we had before) and it just gets regenerated every so often automatically not too different from a package-lock.json.

This is just a draft / WIP test to see if it would work before we do that.